Raspi/Test/mqtt_wireless_log.py

```diff
@@ -0,0 +1,152 @@
+import paho.mqtt.client as mqtt
+import os
+import pandas as pd
+from DataToTempCSV import DataToTempCSV
+import json
+from datetime import datetime
+
+
+def on_connect(client, userdata, flags, rc):
+    """ When the MQTT client connects to the broker it prints out if it
+    connects successfully and then subscribes to all of the wireless-module
+    topics"""
+
+    if rc == 0:
+        print("Connected Successfully! Result code: " + str(rc))
+    else:
+        print("Something went wrong! Result code: " + str(rc))
+
+    # Subscribe to all of the wireless module topics
+    client.subscribe("/v3/wireless-module/#")
+
+
+def on_message(client, userdata, msg):
+    """ MQTT callback for when data is sent on the subscribed
+    '/v3/wireless-module/#' topics. The module_id is found by squashing M
+    infront of the module number. eg M1, M2, M3... etc. The module_id is used
+    for identifying what data came from where and is also used for naming the
+    temp files.
+    """
+
+    module_id = "M" + str(msg.topic[20])
```

Comment:
I wouldn't hard code the number 20 for this case. Instead, try using the string `split()` function to parse the topic string via `/`. You should get an array of strings that were separated by the `/`.